DroneInventorySystem/src/main/java/com/digitalojt/web/repository/CategoryInfoRepository.java

```diff
@@ -14,5 +18,12 @@
  */
 @Repository
 public interface CategoryInfoRepository extends JpaRepository<CategoryInfo, Integer>{
-
+	/**
+	 * 引数に合致する分類情報を取得
+	 * 
+	 * @param categoryName
+	 * @return paramで検索した結果
+	 */
+	@Query("SELECT c FROM CategoryInfo c WHERE c.categoryName = :categoryName")
+    List<CategoryInfo> findByCategoryName(@Param("categoryName") String categoryName);
```

Comment:
```suggestion
   List<CategoryInfo> findByCategoryName(String categoryName);
```
今回の場合、@Queryはいらないですかね。
https://spring.pleiades.io/spring-data/jpa/reference/jpa/query-methods.html